repo/packages/O/omsagent/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+ "packagingVersion": "2.0",
+ "name": "msoms",
+ "version": "1.0.0",
```

Comment:
change to ImageVersion-ServiceVersion 
Ex: if docker tag version is 1.2.3, then this version will be 1.2.3-1.0.0